addons/web/static/src/core/browser/feature_detection.js

```diff
@@ -75,3 +75,12 @@ export function hasTouch() {
 export function maxTouchPoints() {
     return browser.navigator.maxTouchPoints || 1;
 }
+
+export async function hasLocalNetworkAccessSupport() {
+    try {
+        await browser.navigator.permissions.query({ name: "local-network-access" });
+        return true;
+    } catch {
+        return false;
+    }
+}
```

Comment:
I see the point of your implementation but, as this feature is *very* bleeding edge and has a very limited browser compatibility (well, as said in your commit, only Chrome in the next stable version or behind a flag...), I wouldn't add this utility in `web`.

I wouldn't mind revisit this position once the feature reaches a better support.